packages/entity-server/src/models/Project.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { ProjectModel as BaseProjectModel, ProjectType as BaseProjectType } from '@tupaia/database';
+import { Model } from './types';
+
+export interface ProjectFields {
+  readonly code: string;
+  readonly entity_hierarchy_id: string;
+}
+
+export interface ProjectType extends BaseProjectType, ProjectFields {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
I keep adding this but I should just make it a rule in our linting rules. I think it's very common to do this when you want to create a specific subtype with set generic arguments, `Express` seems to do it a lot...

Reply:
Yeah please add the rule